tap_twilio/streams.py

```diff
@@ -12,313 +12,125 @@
-                'pagingation': 'root',
+                'pagination': 'root',
                 'children': {
-                    # pylint: disable=line-too-long
-                    # Reference: https://www.twilio.com/docs/sms/api/media-resource#read-multiple-media-resources
-                    'message_media': {
-                        'api_url': 'https://api.twilio.com',
-                        'api_version': '2010-04-01',
-                        'path': 'Accounts/{AccountSid}/Messages/{ParentId}/Media.json',
-                        'data_key': 'media_list',
-                        'key_properties': ['sid'],
-                        'replication_method': 'FULL_TABLE',  # ALL for parent Address
+                    'channels': {
+                        'api_url': 'https://taskrouter.twilio.com',
+                        'api_version': 'v1',
+                        'path': 'Workspaces/{ParentId}/Workers/{ParentId}/Channels',
+                        'data_key': 'channels',
+                        'key_properties': ["sid"],
+                        'replication_method': 'FULL_TABLE',
+                        'replication_keys': ['date_updated'],
                         'params': {},
-                        'pagingation': 'root'
+                        'pagination': 'root'
                     }
                 }
             },
-            # pylint: disable=line-too-long
-            # Reference: https://www.twilio.com/docs/usage/api/usage-record#read-multiple-usagerecord-resources
-            'usage_records': {
-                'api_url': 'https://api.twilio.com',
-                'api_version': '2010-04-01',
-                'path': 'Accounts/{ParentId}/Usage/Records.json',
-                'data_key': 'usage_records',
-                'key_properties': ['account_sid', 'category', 'start_date'],
-                'replication_method': 'INCREMENTAL',  # Filter query
-                'replication_keys': ['end_date'],
-                'bookmark_query_field_from': 'StartDate',  # Daily
-                'bookmark_query_field_to': 'EndDate',
-                'params': {},
-                'pagingation': 'root'
-            },
-            # pylint: disable=line-too-long
-            # Reference: https://www.twilio.com/docs/usage/api/usage-trigger#read-multiple-usagetrigger-resources
-            'usage_triggers': {
-                'api_url': 'https://api.twilio.com',
-                'api_version': '2010-04-01',
-                'path': 'Accounts/{ParentId}/Usage/Triggers.json',
-                'data_key': 'usage_triggers',
+            'workflows': {
+                'api_url': 'https://taskrouter.twilio.com',
+                'api_version': 'v1',
+                'path': 'Workspaces/{ParentId}/Workflows',
+                'data_key': 'workflows',
                 'key_properties': ['sid'],
-                'replication_method': 'INCREMENTAL',  # Fetch ALL, filter results
+                'replication_method': 'FULL_TABLE',
```

Comment:
Same as above. `FULL_TABLE` is not compatible with the `'replication_keys': ['date_updated'],`, and it should be `INCREMENTAL` if the API endpoint is supporting it.